LAMPStack_Install/recipes/mysql.rb

```diff
@@ -0,0 +1,15 @@
+#Install Mysql
+
+
+#mysql_service 'foo' do
+#  port '3306'
+# version '5.5'
+#  initial_root_password 'P@ssword220413'
+#  action [:create, :start]
+#end
+mysqlpass = data_bag_item("mysql", "rtpass.json")
```

Comment:
what is rtpass.json?

Reply:
knife data bag. A data bag is a global variable that is stored as JSON data and is accessible from a Chef server.The knife data bag subcommand is used to manage arbitrary stores of globally available JSON data.
We can select any name for JSON file, here I am given "rtpass" as  "rootpassword"